docs/content/en/docs/references/feedback/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Skaffold Feedback"
+linkTitle: "Skaffold Feedback"
+weight: 300
+---
+
+Your feedback is very important to us to make Skaffold better!
+Sending us feedback is easy.
+
+## Quick HaTS Survey
+Take our [quick 5-question survey](https://forms.gle/BMTbGQXLWSdn7vEs6) to tell us how satisfied you are with
+Skaffold and and what improvements we should make.
+
+## Skaffold Feedback Sessions
+The Skaffold team actively incorporates user feedback as the product evolves.
+We are looking to chat with developers like you to learn about how you use Skaffold and
+anything we can do to make it better suit your needs.
+
+If you’re interested, please provide your contact information below by filling in
+[Skaffold feedback form](https://forms.gle/J5h567ncypY2ziq49).
+
+We'll reach out to schedule a one-hour hangouts session at a time convenient for you. Providing your information does not guarantee that we will contact you. 
+
+
+No preparation is needed on your part.
+Plus, you'll get the chance to help build something you'll love and a token of our appreciation once participation  is complete!
+
+If at any time you wish to opt out of any communications from Skaffold team, please fill our [opt-out form](https://forms.gle/cnfCXotenyUtF92w6)
```

Comment:
I would add "Note: providing your information does not guarantee that we will contact you." at the end here somewhere.